aptos-move/vm-genesis/src/lib.rs

```diff
@@ -399,7 +399,7 @@ fn initialize(
 }
 
 fn initialize_features(session: &mut SessionExt<impl MoveResolver>) {
-    let features: Vec<u64> = vec![1, 2, 5];
+    let features: Vec<u64> = vec![1, 2, 5, 9];
```

Comment:
Can we make these constants with good names to explain what they are?

Reply:
Yeah, I owe @alinush a helper PR. I'll convert it there.

Reply:
Should this perhaps be in `features.rs`? So anybody adding a feature sees it right away.

Perhaps this also related to the failure of the continuous compat test @vgao1996